packages/trade/exchange-client/tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "outDir": "./dist/js",
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
-        "target": "ES2020",
+        "target": "es2015",
         "types": ["node", "mocha"]
     },
     "include": ["src/**/*"]
```

Comment:
Hmm do we still even need this package exchange-client?

Reply:
Since we have react query clients.

Reply:
Not sure, maybe we can keep them alongside react-query clients in case anyone wants to use the regular ones.
But the main thing is that we have to change `target` in all packages from `es2020`  to `es2015`, because it makes no sense in keeping it `es2020` and also can cause bugs and problems.